platform/migration/platform-migratable-api/src/main/java/org/codice/ddf/migration/ExportMigrationEntry.java

```diff
@@ -262,6 +267,8 @@ public default boolean store() {
    *     <code>false</code> otherwise
    * @throws MigrationException if a failure that prevents the operation from continuing occurred
    * @throws IllegalArgumentException if <code>consumer</code> is <code>null</code>
+   * @throws SecurityException if a security manager exists and its <code>checkRead()</code> method
```

Comment:
:question: Wouldn't store need write access?

Reply:
no because we are reading the file only. The writing is done inside the zip so no need for special permissions there as the migratable framework already has that privileges.  Here we are only making sure that if the migratable is getting its hands on the stream that he better have the permissions for that.